Dockerfile

```diff
@@ -95,6 +95,36 @@ ENV PATH="/usr/lib/fsl/5.0:/usr/lib/afni/bin:$PATH"
 COPY .docker/fsl-6.0/bin/topup /usr/share/fsl/5.0/bin/topup
 COPY .docker/fsl-6.0/lib/* /usr/lib/fsl/5.0/
 
+ENV FSLDIR="/opt/fsl-6.0.1" \
+    PATH="/opt/fsl-6.0.1/bin:$PATH" \
+    FSLOUTPUTTYPE="NIFTI_GZ"
+
+RUN apt-get update -qq && \
+    apt-get install -y -q --no-install-recommends \
+           bc \
+           dc \
+           file \
+           libfontconfig1 \
+           libfreetype6 \
+           libgl1-mesa-dev \
+           libglu1-mesa-dev \
+           libgomp1 \
+           libice6 \
+           libxcursor1 \
+           libxft2 \
+           libxinerama1 \
+           libxrandr2 \
+           libxrender1 \
+           libxt6 \
+           python \
+           wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    echo "Downloading FSL ..." && \
+    wget -q http://fsl.fmrib.ox.ac.uk/fsldownloads/fslinstaller.py && \
+    chmod 775 fslinstaller.py && \
+    /usr/bin/python fslinstaller.py -d /opt/fsl-6.0.1 -V 6.0.1 -q
+
```

Comment:
I put this in because the current docker container didn't have `eddy_openmp` which the node is looking for (in terms of that specific command and also parameters). I thought just installing FSL 6 is the best way to go for this, I'm open to changing it to something else that works though.

Reply:
The Nipype interface for `EddyCorrect` (the older command for eddy) might work here instead. `eddy_openmp` introduced better susceptibility distortion correction but that wouldn't be necessary if we use sdcflows for that.

Reply:
Thanks, in that case I can just use that. `EddyCorrect` doesn't need the index file and acqp inputs, so would it be best to scrap those nodes for now and just use `EddyCorrect`?

Reply:
One thing I'm unsure of is whether the old `eddy_correct` can handle multi-shell or high b-value data. But our test case (the travelling human phantoms) is single-shell.

Reply:
I edited it to just use `eddy_correct` for now, but the other functions needed for `eddy_openmp` are still there in case we want to upgrade. Let me know what you guys think.